src/op_handlers/revert.rs

```diff
@@ -34,6 +34,7 @@ pub fn revert(vm: &mut VMState, opcode: &Opcode) -> Result<bool, EraVmError> {
         vm.flag_lt_of = true;
         let previous_frame = vm.pop_frame()?;
         vm.current_frame_mut()?.pc = previous_frame.exception_handler;
+        vm.current_frame_mut()?.gas_left += previous_frame.gas_left;
```

Comment:
For me it is not necessary because it is a single line (I mean there is no check logic as in `decrease_gas`), but seeing that there is a specific function for `decrease_gas` maybe it is more intuitive to have one for `increment_gas`.